modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Requirements/RequirementsOutlet.tsx

```diff
@@ -38,7 +38,7 @@ const RequirementsOutlet = () => {
 	});
 
 	useEffect(() => {
-		setHeaderActions({actions, item: testrayRequirement, mutate});
+		setHeaderActions({actions: [], item: testrayRequirement, mutate});
```

Comment:
It would be better that you make this validation in a form component.
```suggestion
	setHeaderActions({actions, item: testrayRequirement, mutate});
```